packages/loader/src/GLTFLoader.ts

```
@@ -44,10 +44,22 @@
+        const getPromises = context._getPromises;
+        for (let i = 0, n = getPromises.length; i < n; i++) {
+          getPromises[i].cancel();
+        }
+      });
+      context
+        .parse()
+        .then(resolve)
+        .catch((e) => {
+          glTFResource.destroy();
+          reject(e);
+        })
```

ESLint

[error] Insert `;`

(prettier/prettier)